lgc/patch/FragColorExport.cpp

```diff
@@ -618,11 +618,12 @@ Value *LowerFragColorExport::getOutputValue(ArrayRef<llvm::Value *> expFragColor
 
 // =====================================================================================================================
 // Generates the export instructions for all of the generic output of the fragment shader fragEntryPoint.  Returns the
-// last export instruction that was generated.  Returns nullptr if no export instructions were generated.
+// last export instruction that was generated or the new return instruction if a new one was generated.  Returns nullptr
+// if no export or return instructions were generated.
 //
 // @param fragEntryPoint : The fragment shader to which we should add the export instructions.
 // @param insertPos : All new instructions will be inserted before this position.
-CallInst *LowerFragColorExport::AddExportForGenericOutputs(Function *fragEntryPoint, Instruction *insertPos) {
```

Comment:
Same comment as in #786 about passing in an IRBuilder<> (or BuilderBase if you need any of its facilities) rather than insertPos.